inst/qml/ExploratoryFactorAnalysis.qml

```diff
@@ -189,6 +194,7 @@ Form
 				CheckBox { name: "incl_structure";		label: qsTr("Structure matrix")			}
 				CheckBox { name: "incl_correlations";	label: qsTr("Factor correlations")		}
 				CheckBox { name: "incl_fitIndices";		label: qsTr("Additional fit indices")	}
+				CheckBox { name: "incl_PAtable";		  label: qsTr("Parallel analysis")	}
```

Comment:
```suggestion
CheckBox {
					name:	"incl_PAtable";
					label:	qsTr("Parallel analysis")
					RadioButtonGroup
					{
						name:   "parallelMethodTable"
						title:  qsTr("")

						RadioButton
						{
							value:      "pc"
							label:      qsTr("Based on PC")
							checked:    true
						}
						RadioButton
						{
							value: "fa"
							label: qsTr("Based on FA")
						}
					}
				}
```
I think it makes sense to add this extra option here. It might help users to see the output of the PA with different methods even though they do not want to change the number of factors. What do you think?

Reply:
Was actually thinking about this when testing the module with a dataset. Sounds great!